sample_configs/config.toml

```diff
@@ -0,0 +1,40 @@
+api-host = 'bootstrap.production.bacalhau.org'
```

Comment:
Is it possible to generate this file with comments for each key? In sample config files of yore, everything was commented so that you could open the file and read the documentation inline for the config value you are changing.

Lets also add a CI step to make sure this is up to date like we do for Swagger?

Reply:
I'm not away of any feature in viper that generates config comments, that might have to come later.